backend/pkg/scheduler.go

```diff
@@ -118,18 +119,57 @@ func createReports() {
 
 	qr, err := api.QueryFromResponse(response)
 
-	f, e := excelize.OpenFile("./data/test.xlsx")
+	// Open the template
+	f, e := excelize.OpenFile("./data/template.xlsx")
+
+	// Regexp which matches a cell reference [TODO: Handle $ refs]
+	reg := regexp.MustCompile("([A-Za-z]+)|([0-9]+)")
+
+	// Search for the {title} cell to add into the template
+	result := f.SearchSheet("Sheet1", "{{title}}")
+	f.SetCellValue("Sheet1", result[0], "This is a dynamically added title")
+
+	// Same for the date
+	result = f.SearchSheet("Sheet1", "{{date}}")
+	f.SetCellValue("Sheet1", result[0], "This is a dynamically added date")
+
+	// Find where to start the rows from
+	rowStart := f.SearchSheet("Sheet1", "{{rows}}")
+	split := reg.FindAllString(rowStart[0], 2)
+	rowIndex, _ := strconv.Atoi(split[1])
```

Comment:
Does ``{{rows}}`` have a special meaning here, or something hardcoded into the excel doc?

Reply:
just a symbol to be put into templates to signal where rows should go